doc/source/conf.py

```diff
@@ -115,7 +113,7 @@
 # The short X.Y version.
 version = larray.__version__
 # The full version, including alpha/beta/rc tags.
-release = larray.__version__
+release = 'documentation' #larray.__version__
```

Comment:
Out of curiosity, why do you need this change?

Reply:
read the commit message